backend/src/app/auth.gleam

```diff
@@ -0,0 +1,34 @@
+import birl
+import gleam/json
+import gleam/result
+import gwt
+import simplifile
+import wisp
+
+pub fn generate_jwt(userid: Int, username: String) {
```

Comment:
userid is a String as well. I am not sure why they chose such a confusing name instead of just calling a username.